### PR TITLE
Responsive Footer, Minor UI Changes on Homepage

### PR DESCRIPTION
**Pull request by ajajaj2807**

### CODE REVIEW

**Comment by rakaar** [comment]
Things look really great now
But @ajajaj2807  there are few more important changes as suggested by Karthik Tyagi. Please make them also in this PR
1.logo width in navbar 54px in desktop view
2.left and right hand side of the navbar should have equal padding of 100px in desktop view
3. Nav icons should not be in upper case, with styling in semi bold and size of 24px in desktop view
4. The IIT Tech ambit logo in the page(not in nav) should be center aligned with the text below with size 100px
5. Body in font should be `Montessarrat`
6.  And input for mail was to be kept as it is in the video

File: src/views/Components/Components.scss [hunk 1]
```diff
@@ -2,6 +2,67 @@
   margin-top: 9% !important;
```
Comment (thealphadollar) [inline src/views/Components/Components.scss:2]: Why has `!important` been included everywhere? Is there any other way without enforcing it, this is not a good practice?

File: src/components/Footer/OldFooter.scss [hunk 2]
```diff
@@ -100,8 +100,9 @@
 
 @media only screen and (max-width: 968px) {
   .footer-w {
+    height: max-content !important;
```
Comment (thealphadollar) [inline src/components/Footer/OldFooter.scss:103]: Same issue here.

File: src/index.js [hunk 3]
```diff
@@ -30,7 +30,6 @@ ReactDOM.render(
       <Route path='/issues' component={Issues} />
       <Route exact path='/' component={Components} />
       <Route path='/' component={Footer} />
-      <Route path='/' component={MobileFooter} />
```
Comment (thealphadollar) [inline src/index.js:33]: Please put a screenshot for this.

File: src/views/Components/Components.js [hunk 4]
```diff
@@ -120,16 +121,16 @@ export default function Components(props) {
...
                 </h3>
                 <br />
-                <h3 className={classes.subtitle}>
+                <h3 className={`${classes.subtitle} landing-subtitle2-after`}>
                   Can't Wait ?{' '}
-                  <a href='/stories'>Checkout our latest stories here!</a>
+                  <a href='/stories'>Check out our latest stories here!</a>
                 </h3>
               </Fragment>
             )}
```
Comment (thealphadollar) [inline src/views/Components/Components.js:136]: Please put screenshot for these changes. @ajajaj2807

**Review by thealphadollar — CHANGES_REQUESTED** [review]
Looks really good to me. A few changes and screenshots and we will be good to go.

Thank you for the PR, @ajajaj2807

**Pull request merged**